src/views/IssueChild.js

```diff
@@ -75,12 +75,19 @@ export default function IssueChild() {
 
   const validators = useMemo(
     () => [
-      validateInSet(
-        availablePointNames.getOrElse(new Set()),
-        'This point cannot be spawned.'
-      ),
+      m =>
+        simpleValidatorWrapper({
```

Comment:
I would move this into `validators.js` and probably not try and chain the validators like this — I find it hard to confirm that this does what we're expecting it to. I expect that when you move it to validators and pattern-match the existing logic the bug will resolve itself.